Solutions/Authomize/Analytic Rules/Refactor_AWS_policy_based_on_activities.yaml

```diff
@@ -0,0 +1,61 @@
+id: 642de064-c67b-4eb7-98bd-3f8cd51f282c
+name: Refactor AWS policy based on activities in the last 60 days
+kind: Scheduled
+description: This is a recommended update to IAM policy on AWS. Review the policy and apply it according to change control process. Authomize will have a recommended policy to be downloaded.
+severity: High
+queryFrequency: 5m
+queryPeriod: 30m
+triggerOperator: gt
+triggerThreshold: 0
+tactics:
+- PrivilegeEscalation
+relevantTechniques:
+- T1078
+query: |-
+  Authomize_v2_CL
+  | extend EventID = id_s
+  | extend Policy = policy_name_s
+  | extend Severity = severity_s
+  | extend Description = description_s
+  | extend Recommendation = recommendation_s
+  | extend URL = url_s
+  | extend Tactics = tactics_s
+  | where Policy contains "Refactor AWS policy based on activities in the last 60 days."
+  | project  EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics
```

Comment:
Can we summarize results like below?
| summarize Count=count(), EventEndTime=max(TimeGenerated), EventStartTime=min(TimeGenerated) by EventID, Policy, Severity, Description, Recommendation, URL, Category, Tactics

Reply:
I'll look into this - but again - not sure it matters right now?